bang/magicollections.py

```diff
@@ -1944,14 +1943,15 @@ def to_fields(self, item, *args, **kwargs):
                 diff_value = ''
                 diff = getattr(item, u'{}_difficulty'.format(difficulty), None)
                 if diff is not None:
-                    diff_value = generateDifficulty(diff, True)
-                if getattr(item, u'{}_notes'.format(difficulty), None) != None: diff_value+=_(u'{} notes').format(getattr(item, u'{}_notes'.format(difficulty), None))
+                    diff_value = mark_safe(u'{}<br />'.format(generateDifficulty(diff)))
+                if getattr(item, u'{}_notes'.format(difficulty), None) is not None:
+                    diff_value+=_(u'{} notes').format(getattr(item, u'{}_notes'.format(difficulty), None))
                 if diff_value != '':
                     fields[difficulty] = {
                         'verbose_name': verbose_name,
                         'type': 'html',
-                        'value': diff_value,
-                        'image': staticImageURL('songs/{difficulty}.png'.format(difficulty=difficulty)),
+                        'value': mark_safe(u'{}'.format(diff_value)),
```

Comment:
`u'{}'.format(string)` is equivalent to `string` so:

```python
mark_safe(diff_value)
```

Reply:
✅